hack/csv-generator.go

```diff
@@ -177,10 +184,35 @@ func replaceVariables(flags generatorFlags, csv *csvv1.ClusterServiceVersion) er
 				updatedVariables = append(updatedVariables, envVariable)
 			}
 			updatedContainer.Env = updatedVariables
-			csv.Spec.InstallStrategy.StrategySpec.DeploymentSpecs[0].Spec.Template.Spec.Containers[i] = updatedContainer
+			templateSpec.Containers[i] = updatedContainer
+			if flags.removeCerts {
+				updatedVolumeMounts := templateSpec.Containers[i].VolumeMounts
+				for j, volumeMount := range templateSpec.Containers[i].VolumeMounts {
+					if volumeMount.Name == "cert" {
+						updatedVolumeMounts = append(templateSpec.Containers[i].VolumeMounts[:j], templateSpec.Containers[i].VolumeMounts[j+1:]...)
+						break
+					}
+				}
+				templateSpec.Containers[i].VolumeMounts = updatedVolumeMounts
+			}
 			break
 		}
 	}
+
+	if flags.removeCerts {
+		updatedVolumes := templateSpec.Volumes
```

Comment:
Same here, would it be possible to move this into a separate function?

Reply:
Done, pushed in a separate commit

Reply:
Thanks!
/lgtm